tests/acceptance/SubscriberManageImportExportCest.php

```diff
@@ -101,6 +101,21 @@ private function chooseListAndConfirm(\AcceptanceTester $i) {
     $i->waitForText('Import again');
   }
 
+  private function createNewListAndConfirm(\AcceptanceTester $i) {
+    $newListName = 'Simple List';
+    // first create a new list
+    $i->click('a.mailpoet_create_segment');
+    $i->fillField('input#new_segment_name', $newListName);
+    $i->fillField('textarea#new_segment_description', 'This is just a simple list.');
+    $i->click('input#new_segment_process');
+    // trigger dropdown to display selections and search for recently created list
+    $i->waitForElementVisible('input.select2-search__field');
+    $i->fillField('input.select2-search__field', $newListName);
+    $i->click(['xpath' => '//*[@id="select2-mailpoet_segments_select-results"]/li[1]']);
+    $i->click('.mailpoet_data_manipulation_step [data-automation-id="import-next-step"]');
```

Comment:
You don't need to combine those `[data-automation-id="import-next-step"]` should be enough. We add those data-automation-id attributes to have stable selectors for acceptance tests.

Reply:
I agree, I just copy/paste that code from existing, so I now changed both mine and another inside. Thanks.